components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -97,7 +97,6 @@ - (instancetype)initWithFrame:(CGRect)frame {
 
     self.messageTextView = [[MDCNonselectableTextView alloc] initWithFrame:CGRectZero];
     self.messageTextView.textAlignment = NSTextAlignmentNatural;
-    self.messageTextView.textContainerInset = UIEdgeInsetsZero;
```

Comment:
The default UITextView.textContainerInset is top = 8, bottom = 8.

So I try to decrease the top and bottom of the MDCAlertControllerView.contentScrollView by 8.
If the MDCAlertController only has a message and no title, I need to decrease contentInsets.top and contentInsets.bottom.

If the MDCAlertController only has a message and a title, I need to decrease titleInsets.bottom and contentInsets.bottom. (MDCAlertControllerView will ignore contentInsets.top if title is shown)